notificationExtension/NotificationService.swift

```diff
@@ -9,7 +9,7 @@ import UserNotifications
 
 class NotificationService: UNNotificationServiceExtension {
 
-    var contentHandler: ((UNNotificationContent) -> Void)?
+	var contentHandler: ((UNNotificationContent) -> Void)?
     var bestAttemptContent: UNMutableNotificationContent?
```

Comment:
<table data-meta="generated_by_danger">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/ClearKeep/ck-ios/blob/c7de591c5ac52dc5fd9f730daf7e3ef70907b51c/notificationExtension/NotificationService.swift#L13"></span>Please use tabs instead of spaces<br />
<code>not_spaces</code> <code>NotificationService.swift:13</code></td>
    </tr>
  </tbody>
</table>